linebot/models/base.py

```diff
@@ -90,8 +90,7 @@ def as_json_dict(self):
 
             elif hasattr(value, 'as_json_dict'):
                 data[camel_key] = value.as_json_dict()
-
-            else:
+            elif value is not None:
```

Comment:
I changed this behavior. Do not include null value by default.
It may not break the existent code(I hope :) ).